book/04-git-server/sections/setting-up-server.asc

```diff
@@ -96,15 +96,15 @@ To do so, you must first add the full pathname of the `git-shell` command to `/e
 [source,console]
```

Comment:
There's a typo in the PR title and commit message, the word `deprected` should be `deprecated`. 😉